docs/pages/guides/using-graphql.mdx

```diff
@@ -139,9 +139,3 @@ You can read more about the core GraphQL constructs [here](https://www.howtograp
 
 Learn how to build a GraphQL server, check out this [tutorial](https://www.howtographql.com/graphql-js/0-introduction/).
 
```

Comment:
```suggestion
## Next steps and resources

**GraphQL server basics** &mdash; check out this tutorial series about GraphQL servers:
  - [GraphQL Server Basics (Part 1): The Schema](https://www.prisma.io/blog/graphql-server-basics-the-schema-ac5e2950214e)
  - [GraphQL Server Basics (Part 2): The Network Layer](https://www.prisma.io/blog/graphql-server-basics-the-network-layer-51d97d21861)
  - [GraphQL Server Basics (Part 3): Demystifying the info Argument in GraphQL Resolvers](https://www.prisma.io/blog/graphql-server-basics-demystifying-the-info-argument-in-graphql-resolvers-6f26249f613a)
```

These links are now hosted on Prisma's blog and listed on [howtographql.com](https://www.howtographql.com/basics/2-core-concepts/). Thus, adding this section back with updated links.